float32 AXIS_LEFT_STICK_X
float32 AXIS_LEFT_STICK_Y
float32 AXIS_RIGHT_STICK_X
float32 AXIS_RIGHT_STICK_Y
float32 AXIS_R2
float32 AXIS_L2


bool BUTTON_SQUARE
bool BUTTON_CROSS
bool BUTTON_CIRCLE
bool BUTTON_TRIANGLE

bool BUTTON_L1
bool BUTTON_R1
bool BUTTON_L2
bool BUTTON_R2

bool BUTTON_SHARE
bool BUTTON_OPTIONS

bool BUTTON_LEFT_STICK
bool BUTTON_RIGHT_STICK

bool BUTTON_PS
bool BUTTON_PAD

bool RE_SQUARE
bool RE_CROSS
bool RE_CIRCLE
bool RE_TRIANGLE

bool RE_L1
bool RE_R1
bool RE_L2
bool RE_R2

bool RE_SHARE
bool RE_OPTIONS

bool RE_LEFT_STICK
bool RE_RIGHT_STICK

bool RE_PS
bool RE_PAD

int8 HAT_X
int8 HAT_Y
